app/frontend/packs/application.js

```diff
@@ -20,7 +20,7 @@ const svgs = require.context('../svgs', true)
 const svgPath = (name) => svgs(name, true)
 
 // Tailwind.
-import './stylesheets.css'
+import './../styles/application.css'
```

Comment:
The `packs` directory is meant for webpack entrypoint files only. `application.css` is imported in `application.js`.
Also renamed `stylesheets.css` to `application.css`.